src/java.desktop/share/classes/java/beans/Beans.java

```diff
@@ -103,7 +103,8 @@ public static Object instantiate(ClassLoader cls, String beanName) throws IOExce
      * @throws IOException if an I/O error occurs.
      * @since 1.2
      */
-    @SuppressWarnings("deprecation")
+    @Deprecated(since = "23", forRemoval = true)
```

Comment:
In other places, you didn't add spaces around `=` and used the reversed order of `since` and `forRemoval` attributes.

In majority of other files in this PR, `@SuppressWarnings` precedes `@Deprecated`.